src/bindings/glib/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ else()
 	target_compile_options (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_CFLAGS_OTHER})
 
 	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC elektra-core elektra-kdb)
-	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LDFLAGS})
+	target_link_libraries (${GELEKTRA_LIBRARY} PUBLIC ${GOBJECT_LIBRARIES})
```

Comment:
We dont have yet a `findGObject.cmake` only a `FindGObjectIntrospection.cmake` for`gi`.
The Elektra GLib bindings currently use [pkg-config module](https://cmake.org/cmake/help/v3.5/module/FindPkgConfig.html)
using
`${GOBJECT_LDFLAGS}` should be equal to `pkg-config gobject-2.0 --libs`

I really do not understand yet why this should pull in `-lpthread` or `-fPIE`, but i am no expert of cmake yet.